crypto/keys/secp256k1/secp256k1.go

```diff
@@ -10,7 +10,7 @@ import (
 
 	"github.com/cometbft/cometbft/crypto"
 	secp256k1 "github.com/decred/dcrd/dcrec/secp256k1/v4"
-	"golang.org/x/crypto/ripemd160" //nolint: staticcheck // keep around for backwards compatibility
```

Comment:
I wasn't aware that the SDK is copying over the crypto lib files, rather than importing them... I will try to talk to them

Reply:
Something to report on this during this week's meeting (adding an agenda item)

Resolved